.cirrus.yml

```
@@ -19,7 +19,7 @@ task:
   depends_on: Test
   container:
     image: goreleaser/goreleaser:latest
-  release_script: goreleaser --snapshot
+  release_script: goreleaser --build --snapshot
```

My bad, there was a typo:

```suggestion
  release_script: goreleaser build --snapshot
```